src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -174,16 +174,21 @@
       <header>
         <h2>Amalgamation Statement</h2>
         <p class="mt-4">
-          [*** TODO: blurb ***]
+          Please indicate the statement applicable to this amalgamation.
         </p>
       </header>
 
       <v-card
```

Comment:
You don't need the v-card here. It was a placeholder for your new component.

Right now you have a v-card inside a v-card, which is causing some spacing and error validation issues. (It's not terrible to have a v-card inside a v-card, but it's not needed here. The other components on this page don't do it.)

Reply:
Looks good.